cats_frontend/src/app/modules/cats/components/modals/create-comment/create-comment.component.ts

```diff
@@ -32,19 +28,17 @@ export class CreateCommentComponent implements OnInit {
         this.cat = this.config.data.cat;
     }
 
-    add() {
```

Comment:
missing return type ( everywhere in other file too ) 

- type void if it doesn't return anything